data/builtins/thingengine.builtin/manifest.tt.in

```diff
@@ -196,9 +196,23 @@ class @org.thingpedia.builtin.thingengine.builtin
   #[confirm=false];
 
   action alert()
-  #_[result="Time's up!"]
+  #_[result="Alert"]
```

Comment:
Btw, the agent behavior for this will be:
- says "Notification from Alert: alert" (the first alter is the program name, the second is the result phrase)
- plays the sound

I'm not sure this is what we want.

Reply:
Perhaps we don't need a result phrase or function in thingengine.builtin.ts, and can just play the sound